http/oidc/src/test/java/org/wildfly/security/http/oidc/LogoutConfigurationOptionsTest.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2024 Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.wildfly.security.http.oidc;
+
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+/*
+    Verify that invalid logout config options values are flagged.
+ */
+public class LogoutConfigurationOptionsTest {
+    private OidcJsonConfiguration oidcJsonConfiguration;
+
+    @Before
+    public void before() {
+        oidcJsonConfiguration = new OidcJsonConfiguration();
+        // minimum required options
+        oidcJsonConfiguration.setRealm("realm");
+        oidcJsonConfiguration.setResource("resource");
+        oidcJsonConfiguration.setClientId("clientId");
+        oidcJsonConfiguration.setAuthServerUrl("AuthServerUrl");
+    }
+
+    @After
+    public void after() {
+        System.clearProperty(Oidc.LOGOUT_PATH);
+        System.clearProperty(Oidc.LOGOUT_CALLBACK_PATH);
+        System.clearProperty(Oidc.POST_LOGOUT_URI);
+    }
+
+    @Test
+    public void testLogoutPath() {
+
+        try {
+            System.setProperty(Oidc.LOGOUT_PATH, " ");
```

Comment:
System property related tests should be for the maintenance branch. They shouldn't be present in this 2.x branch.

Reply:
done